projects/storefrontlib/src/cms-structure/services/cms-components.service.ts

```diff
@@ -129,8 +129,9 @@ export class CmsComponentsService {
    */
   getModule(componentType: string): NgModuleRef<any> | undefined {
     return (
-      this.featureModules.hasFeatureFor(componentType) &&
-      this.featureModules.getModule(componentType)
+      (this.featureModules?.hasFeatureFor(componentType) &&
+        this.featureModules?.getModule(componentType)) ||
+      undefined
```

Comment:
Do we need the explicit `undefined` here? Wouldn't that be a result of optional chaining failing?

Reply:
The optional values being `&&` together produces the result of `false` instead of undefined. So this falls back to `undefined` in the result of `false`.